components/Dialogs/EstimateChangingDialog.ts

```diff
@@ -0,0 +1,26 @@
+import { Modal } from '@geist-ui/react';
+import { Story } from '../../redux/types';
+
+interface EstimateChangingDialogParams {
+  story: Story;
+  open: boolean;
+  onClose: Function;
+}
+
+const EstimateChangingDialog = ({ story, open, onClose }: EstimateChangingDialogParams): JSX.Element => {
+  return (
+    <Modal open={open} onClose={onClose}>
+      <Modal.Title>Change Story Estimate</Modal.Title>
+      <Modal.Subtitle>{story.name}</Modal.Subtitle>
+      <Modal.Content>
+        <p>Some content contained within the modal.</p>
+      </Modal.Content>
+      <Modal.Action passive onClick={onClose}>
+        Cancel
+      </Modal.Action>
+      <Modal.Action>Submit</Modal.Action>
+    </Modal>
+  );
+};
+
+export default EstimateChangingDialog;
```

Comment:
This component is the one that's breaking the build. But I need it to get it working so I can share the estimates logic in both `[id]ProjectScreenn` and `StoryCard`.

Reply:
You're gonna hate this :) but the extension needs to be .tsx instead of .ts. I've updated the name to include .tsx and also renamed to EstimateChangeDialog for proper english.

Reply:
Thanks a lot for catching that. I was getting depressed by the linter.